### PR TITLE
WIP: Rerun openapi-gen with kube-openapi #322.

### PR DESCRIPTION
**Pull request by benluddy**
For https://github.com/kubernetes/kube-openapi/pull/322, to verify that no diff is produced in the generated spec.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -539,7 +538,7 @@ replace (
 	k8s.io/kms => ./staging/src/k8s.io/kms
 	k8s.io/kube-aggregator => ./staging/src/k8s.io/kube-aggregator
 	k8s.io/kube-controller-manager => ./staging/src/k8s.io/kube-controller-manager
-	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1
+	k8s.io/kube-openapi => github.com/benluddy/kube-openapi v0.0.0-20220929174216-3b907d108da1
```
Comment (dims) [inline go.mod:541]: personal repo?

Comment (benluddy) [reply go.mod:541]: Yes. This PR is only to demonstrate that in-tree openapi-gen output isn't changed by the related kube-openapi PR (not to actually be merged).

Comment (BenTheElder) [reply go.mod:541]: /hold

**Comment by dims** [comment]
If you still need this PR then please rebase, if not, please close the PR

**Comment by dims** [comment]
This PR has the label work-in-progress, please revisit to see if you still need this, please close if not

**Pull request closed without merging**